src/Microsoft.DotNet.Interactive.Jupyter.Tests/MagicCommandTests.who_and_whos.cs

```diff
@@ -85,8 +85,8 @@ public async Task who_lists_the_names_of_variables_in_scope(Language language)
             {
                 using var baseKernel = language switch
                 {
-                    Language.CSharp => new CSharpKernel().UseWho() as Kernel,
-                    Language.FSharp => new FSharpKernel().UseWho(),
+                    Language.CSharp => new CSharpKernel().UseWho().UseWhos() as Kernel,
```

Comment:
I don't think end users want to specify these separately. Do these need to be separate methods?

Reply:
sorry I forgot to revert